raven/src/main/java/com/getsentry/raven/Raven.java

```diff
@@ -40,9 +59,7 @@
     private final ThreadLocal<RavenContext> context = new ThreadLocal<RavenContext>() {
```

Comment:
These ThreadLocal contexts are already created on a per-instance basis. Thus we only need to be able to access all active Raven instances and retrieve them from those.